GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,89 @@ influence the speed of HP attrition and if it outweighs the benefits
+in the shared inventory stash.
+The stashed items are exactly stashed, they are not simply laying on
+the ground (although the stash has a particular location on the map). 
+To operate the stashed items, you should use commands that refer to 
+the party shared inventory, not to items on the ground.
+
+Obviously, if the item is beneficial (a piece of armor, weapon, or a bonus
+item), it is reasonable to put it into a particular character's outfit.
+However, you should remember that the outfit is limited to any 10 (ten)
+items only. On the contrary, if the item is not required directly (ranged
+weapon, medkits, ampoules with drugs, food, or else), it may be stored in
+the shared inventory stash.
+
+The shared inventory stash is of unlimited size. It has a physical location on
+the map and may be looted. So, it is wise to keep it safe or change 
+its location depending on the current situation. It would be a pity to lose 
+valuables or powerful projectiles by not taking care of the shared inventory. 
+
+To relocate the shared inventory stash, first, you need to remove ALL 
+of the items from it anywhere you want to make it empty. Then, any stashed
+item (or a pile of items) will automatically become the new stash. The process 
+takes a couple of turns, so you should plan shared inventory stash relocation
+in advance.
+You should keep your eye on the item management stat in the character menu
+`#`. If this stat is too low (which may be caused by an injury or low 
+calmness level), it will be difficult for the character to work with items in 
+the character's personal outfit or in the shared inventory stash.
+
+To open the pointman's outfit, press `O` (if you want to see the pointman's
+personal inventory outfit and manipulate items stored there) or `i` (if
+you want to stash something in the shared inventory from the pointman's
+outfit).
+To open the shared inventory stash, press `I`.
+To open the whole team's belongings (personal and stashed), press `T`.
+To manage items on the ground (trigger or fling it directly not putting
+it into any inventory to save time), press `G`.
+
+To pick up an item from the ground to the pointman's outfit, press `g`
+or `,`. Items that have some direct use (melee weapon or armor) 
+will be equipped in the personal equipment outfit while if it has no 
+direct use or the equippment outfit is full, the item will be moved
+to the shared inventory.
+
+When you are managing items in the inventory or on the ground, you should
+choose the item with the LMB or arrows and `RET` buttons to get the item
+management menu. You may want to choose all items from the particular 
+location by pressing `*`.
+
+You may want to cycle among the characters' inventory outfits without leaving 
+the inventory menu. To do that, you may press `<` or `>` to check different 
+inventories: character's outfit, stashed inventory, objects on the ground, etc. 
+You may also change pointmen that are adjusting inventory by pressing 
+the `TAB` command.
```

Comment:
Considering the fact that this command caused the game to crash, is it reasonable to keep it functional? Maybe, it is better to disable it? Personally me, I have never used it.

Reply:
I'm afraid, to disable it, we'd need to be able to compile the game and release a new version. But it's already fixed (I mean the crash, the occasional lack of any effect is not, if that's a bug) in the new version. However, feel free to remove the sentence describing it. It's quite obvious what it does, so no point keeping it, especially that it's broken.

Reply:
Well, I do not mind against this fragment. Shall I keep it?

Reply:
Delete it. It's, in some form, in in-game help anyway.

Reply:
done.